--- a/env.py
+++ b/env.py
@@ -966,1 +966,0 @@
-        self.prev_residual = target_handle_angle
@@ -979,2 +978,1 @@
-        reward = tracking_reward + bonus_reward + smoothness_penalty + improvement_reward + action_penalty + residual_penalty
-        reward = tracking_reward + bonus_reward + smoothness_penalty + improvement_reward + action_penalty + subgoal_reward
+        reward = tracking_reward + bonus_reward + smoothness_penalty + improvement_reward + action_penalty + residual_penalty + subgoal_reward